README.md

```diff
@@ -209,4 +209,50 @@ Logging has been set up to store logs at info level and above. Logs are stored a
 - Error logs set up to record if tasks do not behave as expected e.g. does not change status to started when the corresponding button is clicked.
 -Warning logs set to record the user ID when an authorised user logs into the app, when a new user visits the app or when a user with no write access tries to edit the tasks
 
+## Minikube
+The todo app runs locally using minikube. To run the app via Minikube run the following commands:
+
+- minikube start
+This will start the minikube cluster
+
+- kubectl port-forward service/todo-app 5000:8000
```

Comment:
Worth noting that this assumes the user has already:
* Built the image & loaded it into Minikube
* Applied the deployment & service & secret (which are mentioned later, but aren't obviously requirements here)